beacon_node/execution_layer/src/lib.rs

```diff
@@ -194,6 +198,8 @@ pub enum BlockProposalContents<E: EthSpec, Payload: AbstractExecPayload<E>> {
         kzg_commitments: KzgCommitments<E>,
         /// `None` for blinded `PayloadAndBlobs`.
         blobs_and_proofs: Option<(BlobsList<E>, KzgProofs<E>)>,
+        // TODO(pawan): this should probably be a separate variant/superstruct
```

Comment:
Maybe we change this to `TODO(electra)` before merging? Or do you want to fix it first?

```suggestion
        // TODO(electra): this should probably be a separate variant/superstruct
```